src/modules/ArbitraryCrop/Module.js

```diff
@@ -0,0 +1,89 @@
+module.exports = function ArbitraryCrop(options, UI, util) {
+  var defaults = require('../../util/getDefaults.js')(require('./info.json'));
+
+  options.x1 = options.x1 || defaults.x1;
+  options.y1 = options.y1 || defaults.y1;
+  options.x2 = options.x2 || defaults.x2;
+  options.y2 = options.y2 || defaults.y2;
+  options.x3 = options.x3 || defaults.x3;
+  options.y3 = options.y3 || defaults.y3;
+  options.x4 = options.x4 || defaults.x4;
+  options.y4 = options.y4 || defaults.y4;
+
+  var output;
+
+  // This function is called on every draw.
+  function draw(input, callback, progressObj) {
+    progressObj.stop(true);
```

Comment:
Do we still need these? I remember taking out the progress code..

Reply:
I don't exactly know, should I remove this?